### ISSUE TITLE
[RFC] ApiPlatform DataProvider for dictionaries

### ISSUE THREAD

**Issue by Nek-** [biig-io/DictionaryBundle#28]
ApiPlatform still have some drawbacks for supporting dictionaries. https://github.com/api-platform/core/issues/1574 https://github.com/api-platform/core/issues/1584

But we can still override ApiPlatform and try to fix it. So I suggest adding a DataProvider for ApiPlatform.

**Comment by Awkan** [comment]
If we are able to maintain this (temporary ?) fix, we can add it to this bundle.

Do not declare it as service by default and give the steps to follow in documentation if we wanna use it with ApiPlatform.

**Comment by Nek-** [comment]
I don't think we should add any fix related to ApiPlatform inside the bundle. But maybe we can document how to fix it inside applications. (would be great IMHO)

**Comment by Awkan** [comment]
Indeed 👍 
I prefer this idea too
